### ISSUE TITLE
Predicted bounding boxes with half value than the real value

### ISSUE THREAD

**Issue by alberdibu1** [experiencor/keras-yolo2#289]
Hi all!

I have a problem at prediction step in video and image. The predicted bounding box coordinates doesn't fit the trained object (Persons), but if all the predicted coordinates are multiply by two fit the person perfectly. I don't understand why.

My config file:

{
    "model" : {
        "backend":              "Tiny Yolo",
        "input_size":           416,
        "anchors":              [0.59,0.96, 0.73,0.60, 0.93,1.39, 1.06,0.94, 1.36,1.34],
        "max_box_per_image":    10,        
        "labels":               ["Person"]
    },

    "train": {
        "train_image_folder":   "zenith_data/",
        "train_annot_folder":   "",     
          
        "train_times":          8,
        "pretrained_weights":   "",
        "batch_size":           16,
        "learning_rate":        1e-4,
        "nb_epochs":            20,
        "warmup_epochs":        3,

        "object_scale":         5.0 ,
        "no_object_scale":      1.0,
        "coord_scale":          1.0,
        "class_scale":          1.0,

        "saved_weights_name":   "tiny_yolo_zenith.h5",
        "debug":                true
    },

    "valid": {
        "valid_image_folder":   "",
        "valid_annot_folder":   "",

        "valid_times":          1
    }
}

The training works perfectly with good results, but i have this little problem at prediction stage. Any idea?

Thanks!! =)
